cultivation/scripts/flashcore/yaml_processing/yaml_processor.py

```
@@ -300,15 +166,52 @@
+    if not config.skip_media_validation and not config.assets_root_directory.exists():
+        return [], [
+            YAMLProcessingError(
+                file_path=config.assets_root_directory,
+                message=f"Assets root directory does not exist: {config.assets_root_directory}",
+            )
+        ]
 
-    Args:
-        config: A YAMLProcessorConfig object holding all configuration parameters.
+    yaml_files = sorted(list(config.source_directory.rglob("*.yaml"))) + \
+                 sorted(list(config.source_directory.rglob("*.yml")))
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)